src/about.js

```diff
@@ -1,35 +1,9 @@
-const { shell, BrowserWindow } = require('electron');
+const { shell } = require('electron');
```

Comment:
For longer-term talk: I'm not clear why we have `about.js` vs `about_dialog.js`.  Can we merge the two?

Reply:
The idea of two files is to separate what runs in the render process from the main process. It's easy to get conflicts and think you are spanning them so I tend to keep them separate.